diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter4/NullChecksRule.kt

```diff
@@ -92,12 +126,27 @@ class NullChecksRule(configRules: List<RulesConfig>) : DiktatRule(
         val elseCodeLines = condition.extractLinesFromBlock(ELSE)
         val text = if (isEqualToNull) {
             when {
-                elseCodeLines.isNullOrEmpty() -> "$variableName ?: run {\n${thenCodeLines?.joinToString(separator = "\n")}\n}"
+                elseCodeLines.isNullOrEmpty() ->
+                    if (condition
+                        .treeParent
+                        .findChildByType(THEN)?.let {
+                            it.findChildByType(BLOCK) ?: it
+                        }?.hasChildOfType(BREAK) == true
```

Comment:
I have seen this code here already, you should extract it to a function, like `getBreakNodeFromIf`